YIF.Core.Data/Entities/SpecialtyToIoEDescription.cs

```diff
@@ -4,20 +4,19 @@ namespace YIF.Core.Data.Entities
 {
     public enum PaymentForm
     {
-        Day,
-        //Сorrespondence,
-        Evening
+        Budget,
+        Contract
     }
     public enum EducationForm
     {
-        Budget,
-        Contract
+        Day,
+        Evening
```

Comment:
I think that we have daily and external forms of education for MVP. Maybe it's need discussion